tests/functional/pv/pv_services/test_pvc_acceptance.py

```diff
@@ -0,0 +1,828 @@
+# tests/functional/pv/pv_services/test_dynamic_pvc_accessmodes_with_reclaim_policies.py::TestDynamicPvc
+#     test_rwo_dynamic_pvc[CephBlockPool-Retain]
+#     test_rwo_dynamic_pvc[CephBlockPool-Delete]
+#     test_rwo_dynamic_pvc[CephFileSystem-Retain]
+#     test_rwo_dynamic_pvc[CephFileSystem-Delete]
+#     test_rwx_dynamic_pvc[CephFileSystem-Retain]
+#     test_rwx_dynamic_pvc[CephFileSystem-Delete]
+# tests/functional/pv/pv_services/test_pvc_assign_pod_node.py::TestPvcAssignPodNode
+#     test_rwo_pvc_assign_pod_node[CephBlockPool]
+#     test_rwo_pvc_assign_pod_node[CephFileSystem]
+#     test_rwx_pvc_assign_pod_node[CephBlockPool]
+#     test_rwx_pvc_assign_pod_node[CephFileSystem]
+# tests/functional/pv/pv_services/test_raw_block_pv.py::TestRawBlockPV
+#     test_raw_block_pv[Delete]
+#     test_raw_block_pv[Retain]
+#
+# ## partially implemented/some parameters are different
+# tests/functional/pv/pvc_resize/test_pvc_expansion.py::TestPvcExpand::test_pvc_expansion
+#   - the original test_pvc_expansion performs expansion on 5 PVCs (2 cephfs, 3 rbd)
+#   - some of the configuration of PVCs and maybe also PODs is/might be different
+#
+# tests/functional/pv/pv_services/test_pvc_delete_verify_size_is_returned_to_backendpool.py
+#     test_pvc_delete_and_verify_size_is_returned_to_backend_pool
+
+# 1. Create PVCs according to this table:
+#   Type                        RWO    RWX  Recliam policy
+#   RBD-Filesystemvolume mode   yes    No   delete  storage class default class is delete
+#   CEPHFS                      yes    yes  delete
+#   RBD-Block                   yes    yes  delete
+#   RBD-Filesystemvolume mode   yes    no   retain
+#   CEPHFS                      yes    yes  retain
+#   RBD-Block                   yes    yes  retain
+# 2. Create pods on these PVCs by assigning nodeName to these pods
+# 3. Run IOs. Check data integrity
+# 4. Expand the PVCs and verify that new size is in effect
+# 5. Run IOs. Check data integrity
+# 6. Delete Pods
+# 7. Delete PVCs. For the PVCs with Reclaim policy set to Delete, make sure the PV is deleted
+#    and for those with policy set to retain the PV remains undestroyed
+
+import functools
+
+import logging
+
+# from pytest_check import check
+
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    acceptance,
+)
+
+from ocs_ci.framework import config
+from ocs_ci.framework.pytest_customization.marks import green_squad
+from ocs_ci.helpers import helpers
+from ocs_ci.helpers.helpers import default_storage_class
+from ocs_ci.ocs import constants, node
+from ocs_ci.ocs.cluster import is_managed_service_cluster
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+from ocs_ci.ocs.resources import pod
+from ocs_ci.utility import version
+from ocs_ci.utility.retry import retry
+from ocs_ci.utility.utils import TimeoutSampler
+
+logger = logging.getLogger(__name__)
+
+
+@green_squad
+@acceptance
+class TestPvcAcceptance(ManageTest):
+    """
+    Acceptance tests for PVC
```

Comment:
It would be great if you can share the improvement in the execution time compared to old implementation.

Reply:
I tried to gather some summary from last bunch of executions and the average execution time of the original tests covered by this one cumulative tests takes around 35 minutes (30 - 40 minutes), while few of the executions of the cumulative test takes around 17 minutes (15 - 19 minutes, I saw some executions with 12 and 13 minutes).

But I would not consider this as the fully representative statistics, we will need more executions and maybe also compare the full acceptance tests execution time.